lib/contestant.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './game'
+
+class Contestant < Game
+
+  attr_reader :first_name, :last_name, :age, :state_of_residence, :spending_money, :full_name, :game_interests
+
+  def initialize(info_hash)
+    @first_name = info_hash[:first_name]
+    @last_name = info_hash[:last_name]
```

Comment:
it's fine to capture these here, but nothing in the interaction pattern suggests that we need first and last time, so i'd just leave these out and use the full_name instead.